algorithms/aflfast/aflfast_other_hierarflow_routines.cpp

```diff
@@ -34,7 +34,7 @@ AFLMidCalleeRef<AFLFastState> ApplyDetMutsTemplate<AFLFastState>::operator()(
     // So we can reload the file with mmap.
     testcase->input->LoadByMmap(); // no need to Unload
 
-    if (state.orig_perf && state.queued_paths > 10) {
+    if (!state.orig_perf && state.queued_paths > 10) {
```

Comment:
To make clear the fact that `orig_perf` is an integer variable, I prefer `== 0`  than `!`.

```suggestion
    if (state.orig_perf == 0 && state.queued_paths > 10) {
```